src/Paket.Core/ProjectFile.fs

```diff
@@ -200,17 +200,8 @@ module ProjectFile =
             None
 
     let createNode name (project:ProjectFile) =
-        let namespaceURI = 
-            try
-                project.Document.DocumentElement.NamespaceURI
-            with
-            | _ -> null
-        if String.IsNullOrWhiteSpace namespaceURI then
-            project.Document.CreateElement (name, Constants.ProjectDefaultNameSpace)
-        else
-            project.Document.CreateElement (name, namespaceURI)
+        project.Document.CreateElement (name, project.Document.DocumentElement.NamespaceURI)
```

Comment:
It s working on dotnet core csproj like a charm and tests are passing but 2 who where already down